### DISCUSSION TITLE
What's the best way to suppress the date on a particular page?

### DISCUSSION THREAD

**Discussion by moritzreiter** [hugo-sid/hugo-blog-awesome#78] [Q&A] [answered]
Hi! I'm completely new to Hugo and experimenting with this theme for a blog.

I can't find an obvious way to suppress showing the date on a particular page. I do want to have a date on a blog post, of course, but on an "About" page, it doesn't make a lot of sense to me to show a date.

I was hoping I would be able to suppress the date by removing it from the meta data of the markdown file or with some other kind of config, but I can't find how. Or will I have to modify the theme code to achieve this?

**Comment by hugo-sid** [comment, marked as answer]
Hi @moritzreiter. Welcome to Hugo blog awesome discussions.

Thanks for posting this question.

>  on an "About" page, it doesn't make a lot of sense to me to show a date.

I agree. I have pushed few commits which make it optional to specify the date.

## Solution

> I was hoping I would be able to suppress the date by removing it from the meta data of the markdown file

You have to do exactly this. Simply omit the date parameter.

I observed that you have added this theme as Git module in your [blog](https://github.com/moritzreiter/blog). Please update your Git module to make use of this feature.

## Demo

Markdown file: 
https://github.com/hugo-sid/hugo-blog-awesome/blob/68c1599c16493878126b48d61f69744c5b0a6384/exampleSite/content/pages/about.md?plain=1#L1-L6

Rendered page corresponding to the above Markdown file: https://hba.sid.one/pages/about/

## Conclusion

Let me know if this solution works for you. In case it does, you can mark this as the answer. Otherwise, feel free to post your observations here. 
Don't forget to star the repo, if you like this theme.

**Reply by moritzreiter** [reply]
Works like a charm, thanks a lot! 😊

**Reply by hugo-sid** [reply]
I'm glad it worked. You're welcome @moritzreiter 😊
